test/html.test.js

```diff
@@ -438,6 +443,75 @@ function() {
   QUnit.equal(mp4Segments[0][1], 6, 'fragment contains the correct second byte');
 });
 
+QUnit.test(
+'only appends init segment for first segment or on audio changes',
```

Comment:
I know that audio changes change the appendInitSegment_ value, but I didn't see anything in the test that identifies audio changes.